### ISSUE TITLE
[Bug]: Zooming and scaling must not be disabled

### ISSUE THREAD

**Issue by damienrobson-sage** [storybookjs/storybook#29346]
### Describe the bug

AXE DevTools finds a critical error when ran against Storybook:

```
Ensure <meta name="viewport"> does not disable text scaling and zooming
```

This is due to the `viewport` meta tag in the `<head>` element:

```html
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
```

For accessibility reasons, zooming and scaling should be allowed in order to avoid violation of [1.4.4 Resize Text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html). 

### Reproduction link

https://carbon.sage.com/?path=/story/welcome--welcome-page

### Reproduction steps

1. Open a Storybook
2. Run Axe DevTools
3. Note the zooming and scaling failure

### System

Storybook Environment Info:

  System:
    OS: macOS 14.6.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.18.0 - ~/.nvm/versions/node/v20.18.0/bin/node
    npm: 10.5.2 - ~/git/carbon/node_modules/.bin/npm <----- active
  Browsers:
    Edge: 129.0.2792.89
    Safari: 17.6
  npmPackages:
    @storybook/addon-a11y: ^8.2.6 => 8.2.6 
    @storybook/addon-actions: ^8.2.6 => 8.2.6 
    @storybook/addon-controls: ^8.2.6 => 8.2.6 
    @storybook/addon-docs: ^8.2.6 => 8.2.6 
    @storybook/addon-toolbars: ^8.2.6 => 8.2.6 
    @storybook/addon-viewport: ^8.2.6 => 8.2.6 
    @storybook/addon-webpack5-compiler-swc: ^1.0.4 => 1.0.5 
    @storybook/components: ^8.2.6 => 8.2.6 
    @storybook/manager-api: ^8.2.6 => 8.2.6 
    @storybook/preview-api: ^8.2.6 => 8.2.6 
    @storybook/react: ^8.2.6 => 8.2.6 
    @storybook/react-webpack5: ^8.2.6 => 8.2.6 
    @storybook/theming: ^8.2.6 => 8.2.6 
    @storybook/types: ^8.2.6 => 8.2.6 
    chromatic: ^6.17.4 => 6.24.1 
    storybook: ^8.2.6 => 8.2.6

**Comment by Sidnioulz** [comment]
Thanks for the report @damienrobson-sage! We've fixed this through #31283 that addressed the other issue for this problem: #31272. I'll go ahead and close this issue.
